<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0" columnstretch="1,0,0,0">
    <item row="4" column="3">
     <widget class="QTextBrowser" name="tips">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Tips -&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1. 筛分数据&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2. 两张表格vlookup&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3. 合并一个文件夹内的xlsx和csv文件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ScreenColumn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="columnChoice"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>nil</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="symbolChoice"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leftLineEdit">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rightLineEdit">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="4">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QTableView" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="dispalyDesc">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QTextBrowser" name="fileSavePath">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTextBrowser" name="txtPath">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="openFilePath">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTextBrowser" name="recoedLog"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Screen</string>
    </property>
    <addaction name="actionFile"/>
    <addaction name="actionCheckData"/>
    <addaction name="actionSinglePath"/>
    <addaction name="actionMultiplePath"/>
    <addaction name="actionTxt"/>
    <addaction name="actionSingleScreen"/>
    <addaction name="actionMultipleScreen"/>
   </widget>
   <widget class="QMenu" name="helpBtn">
    <property name="title">
     <string>Vlookup</string>
    </property>
    <addaction name="actionInitTable"/>
    <addaction name="actionMappingTable"/>
    <addaction name="actionMerge"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionLight"/>
    <addaction name="actionDark"/>
   </widget>
   <widget class="QMenu" name="menuMerge">
    <property name="title">
     <string>Merge</string>
    </property>
    <addaction name="actionMergePath"/>
    <addaction name="actionMergeSavePath"/>
    <addaction name="actionMergeFile"/>
   </widget>
   <widget class="QMenu" name="menuPivot">
    <property name="title">
     <string>Pivot</string>
    </property>
    <addaction name="actionPivotWindow"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Visualization</string>
    </property>
    <addaction name="actionPlot"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="helpBtn"/>
   <addaction name="menuMerge"/>
   <addaction name="menuPivot"/>
   <addaction name="menuPlot"/>
   <addaction name="menuStyle"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest3">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="actionTest3_2">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionSinglePath">
   <property name="text">
    <string>SinglePath</string>
   </property>
  </action>
  <action name="actionTxt">
   <property name="text">
    <string>Txt</string>
   </property>
  </action>
  <action name="actionCheckData">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="actionSingleScreen">
   <property name="text">
    <string>SingleScreen</string>
   </property>
  </action>
  <action name="actionVlookup">
   <property name="text">
    <string>Vlookup</string>
   </property>
  </action>
  <action name="actionInitTable">
   <property name="text">
    <string>InitTable</string>
   </property>
  </action>
  <action name="actionMappingTable">
   <property name="text">
    <string>MappingTable</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="text">
    <string>vlookup</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionMergePath">
   <property name="text">
    <string>MergePath</string>
   </property>
  </action>
  <action name="actionMergeFile">
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="actionMergeSavePath">
   <property name="text">
    <string>SavePath</string>
   </property>
  </action>
  <action name="actionMultiplePath">
   <property name="text">
    <string>MultiplePath</string>
   </property>
  </action>
  <action name="actionMultipleScreen">
   <property name="text">
    <string>MultipleScreen</string>
   </property>
  </action>
  <action name="actionPivotWindow">
   <property name="text">
    <string>PivotWindow</string>
   </property>
  </action>
  <action name="actionPivot">
   <property name="text">
    <string>Pivot</string>
   </property>
  </action>
  <action name="actionPlotDemo">
   <property name="text">
    <string>demo</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>VisualWindow</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
